WcaOnRails/config/locales/en.yml

```diff
@@ -68,6 +68,13 @@ en:
     "3": "Best of 3"
     "a": "Average of 5"
     "m": "Mean of 3"
+    #context: short name alternative
+    short:
+      "1": "Bo1"
+      "2": "Bo2"
+      "3": "Bo3"
+      "a": "Ao5"
+      "m": "Mo3"
```

Comment:
Do you think it's likely to differ across languages? Just pointing out that when updating Polish translation I kept the full names in English - they are more popular anyway and there's no way of differentiating between *mean* and *average* using a single world in Polish. That's why I wonder if there's any actual chance of short formats to differ.

Reply:
I actually don't know :s
Long version of formats may surely change across language, but for short names I was also thinking of using the English ones...